pkg/netdevice/netInfoProviders.go

```diff
@@ -72,11 +72,21 @@ func NewVhostNetInfoProvider() types.DeviceInfoProvider {
 /* DeviceInfoProvider Interface */
 
 func (rip *vhostNetInfoProvider) GetDeviceSpecs() []*pluginapi.DeviceSpec {
+	deviceSpec := make([]*pluginapi.DeviceSpec, 2)
+
 	if !VhostNetDeviceExist() {
 		glog.Errorf("GetDeviceSpecs(): /dev/vhost-net doesn't exist")
 		return nil
 	}
-	return GetVhostNetDeviceSpec()
+	deviceSpec[0] = GetVhostNetDeviceSpec()[0]
```

Comment:
why not just 

```
deviceSpec := GetVhostNetDeviceSpec()

...
...

deviceSpec := append(deviceSpec, GetTunDeviceSpec()...)
return deviceSpec
```

Reply:
I just didn't want to append as in the underling it will create a new slice and copy everything but again here is only a small slice so if you want I can change the code :)

Reply:
also, if later on, if either GetVhostNetDeviceSpec() or GetTunDeviceSpec() end up returning more than one element, you will not need to change here :)

Reply:
works for me :)